notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>Zwróć uwagę, że <code>kod_terytorialny_wojewodztwo</code>  i <code>kod_terytorialny_powiat</code>  nie powinny być traktowane jak wartości numeryczne bo to w zasadzie id jednostki administracyjnej. To, że w maksie dla województw mamy 32 to dobry przykład.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
poprawione